jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepExecution.java

```diff
@@ -1053,7 +1060,7 @@ public void expand(EnvVars env) throws IOException, InterruptedException {
 

```

Comment:
You are still extending `BodyExecutionCallback.TailCall`. You need to extend the new inner class `[GeneralNonBlockingStepExecution.]TailCall`.

Reply:
(resolved, but GH is not showing me the resolve button)